db/dcrsqlite/apisource.go

```diff
@@ -1331,6 +1347,20 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 	}
 }
 
+// IsZeroHashP2PHKAddress checks if the given address is the dummy (zero pubkey
+// hash) address. See https://github.com/decred/dcrdata/issues/358 for details.
+func IsZeroHashP2PHKAddress(checkAddressString string, params *chaincfg.Params) bool {
```

Comment:
Please make a `IsZeroHashP2PHKAddress` in apisource_test.go to verify that it returns the expected result for various strings and all three networks.